core/Cargo.toml

```diff
@@ -28,9 +28,11 @@ serde_json = "1.0"
 serde_yaml = "0.8"
 uuid = { version = "0", optional = true }
 thiserror = "1.0.19"
+serde_qs = { version = "0.6", optional = true, features = ["actix"] }
 
 [features]
 actix = ["v2", "actix-http", "actix-web", "pin-project"]
 codegen = ["v2", "heck", "log"]
 nightly = ["paperclip-macros/nightly"]
 v2 = ["paperclip-macros/v2"]
+qs = ["serde_qs"]
```

Comment:
Do we need this? We can directly use dependency names for features. Besides, you're doing exactly that below.

Reply:
Ahh didn't notice you could do that, thanks, updated